test/data/mriqc-example/usage.json

```diff
@@ -0,0 +1,6 @@
+{"timestamp": "2024-10-08T14:37:00.831368-04:00", "num_samples": 1, "processes": {"2703236": {"pcpu": 0.0, "pmem": 0.0, "rss": 40456192, "vsz": 1603104768, "timestamp": "2024-10-08T14:37:00.831368-04:00", "etime": "00:00", "stat": {"Ssl": 1}, "cmd": "singularity run --contain --bind /home/asmacdo/devel/sandbox/mriqc-sanity/sourcedata:/data:ro --bind /home/asmacdo/devel/sandbox/mriqc-sanity/long-run-4:/out --bind /home/asmacdo/devel/sandbox/mriqc-sanity/long-run-4/workdir:/workdir docker://nipreps/mriqc:24.0.2 /data /out participant --participant-label 02 -w /workdir --no-sub"}}, "totals": {"pmem": 0.0, "pcpu": 0.0, "rss": 40456192, "vsz": 1603104768}, "averages": {"rss": 40456192, "vsz": 1603104768, "pmem": 0.0, "pcpu": 0.0, "num_samples": 1}}
```

Comment:
I had planned to replace this with a stripped down json, but on second thought, this isn't really that big-- maybe its best to pass a legit file with real data in it.